cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/CloudLoadBalancerMetadata.java

```diff
@@ -0,0 +1,37 @@
+package com.sequenceiq.cloudbreak.cloud.model;
+
+import com.sequenceiq.common.api.type.LoadBalancerType;
+
+public class CloudLoadBalancerMetadata {
+
+    private final LoadBalancerType type;
+
+    private final String cloudDns;
+
+    private final String hostedZoneId;
+
+    private final String ip;
```

Comment:
The ip field isn't used for AWS load balancers at all. We instead use the dns and hostedZoneId. I put this here because my understanding is that Azure LBs use IPs, so we'll need an ip field eventually. But if it needs to be a list we'll update it when the Azure work is being done.